Source/SwiftLintBuiltInRules/Rules/Style/SwitchCaseAlignmentRule.swift

```diff
@@ -216,5 +227,14 @@ extension SwitchCaseAlignmentRule {
                 """)
             ]
         }
+
+        private func getOneLinerExample(ignoreOneLiners: Bool) -> Example {
+            let marker = ignoreOneLiners ? "" : violationMarker
+
+            return Example(
```

Comment:
I wish there were more examples, e.g. some were the opening and closing braces are on different lines.

Reply:
Good call, I've added some extra tests and tweaked the rule so that it checks for the line of the switch keyword instead of opening brace to determine if it's a one liner, so that this doesn't count as one

```swift
switch i
{ case .x: 1 default 0 }
```